lib/heroku/helpers/addons/api.rb

```diff
@@ -0,0 +1,98 @@
+module Heroku::Helpers
+  module Addons
+    module API
+      VERSION="3.switzerland".freeze
```

Comment:
Okay, I wasn't sure if EVERYTHING was deployed as `3`, yet and figured it would fallback to `3` for the things that were. I'll update this though.